index.html

```diff
@@ -21,7 +21,7 @@
         </nav>
     </header>
     <main>
-        <span class="date">Date:</span>
+        <div class="date"></div>
```

Comment:
- [ ] Great job so far 👏🏻  However as required by the previous reviewer, Kindly change your style file name, rename it from `style.css` to `index.css` see the project requirement  ⬇️ 

![11111](https://user-images.githubusercontent.com/79058364/160474969-6e9e6ba1-4d7e-4525-ae9c-6ba4f4846fe9.png)